### ISSUE TITLE
Label, Button and Span text removing leading Zero from string

### ISSUE THREAD

**Issue by tsonevn** [NativeScript/nativescript-angular#822]
The leading `0` in the Label, Button or Span, while using `FormattedText` will be removed.
For example, if we try to display `016` the result will be `16`. 
The similar behavior could be reproduced with the attached sample project.
[Archive.zip](https://github.com/NativeScript/nativescript-angular/files/1031959/Archive.zip)

**Comment by rrjohnson85** [comment]
For what is is worth, the following elements all render `1` instead of the specified text of `00001`:
```
<Label text="00001"></Label>
<TextField text="00001"></TextField>
<Button text="00001"></Button>
```

**Comment by sis0k0** [comment]
fixed with #806
It's available if using `next` tag and will be released as part of NativeScript Angular 3.1
